server/jetstream_api.go

```diff
@@ -1127,6 +1139,26 @@ func (s *Server) jsStreamRestoreRequest(sub *subscription, c *client, subject, r
 	// Create our internal subscription to accept the snapshot.
 	restoreSubj := fmt.Sprintf(jsRestoreDeliverT, stream, nuid.Next())
 
+	s.Noticef("Starting restore for stream %q in account %q", stream, c.acc.Name)
```

Comment:
Not sure this is a notice, on something like NGS will fill logs no? Maybe debug?

Reply:
I doubt we will see a flood of these kind of snapshot. But someone taking a snapshot is a significant event and something that should be noticed by operators for sure - even as correlation if we see sudden spikes in traffic these logs are important 

So I would keep them notice - at least until we have some kind of even correlation in surveyor

Reply:
These can also have significant IO impact on the server, doing that silently is also not good - we should also log around jsck.

The lack of logs is a long term bug for me in nats server, its really hard to know whats going on since everything is debug - and non of these things are debug.  Debug is for developers, notice is for operators.  Yet NATS Server treats debug as operational feedback.

Reply:
If account owners could see all of these for their account would solve the problem vs operators. Operators will care about high level stuff but not your individual issues IMO. Assuming we can make the stream -DV system service work I would still suggest debug.

Reply:
Operations people have tools like logz.io and logstash and you know from even our internal use that we lean on this quite heavily. It’s the way people operate software and we should enable them to operate software the way the industry does it. 

Yes we need the logging advisory service but that’s in addition to not to replace this. The people running the servers don’t have insight into each accounts operation these log lines are for people running the service. 

With yet more debug level logs if suddenly there is a increase in I/O there is literally no hint at all about what’s going on to people operating the service even with the advisories since they are account level and so not visible to the operators.

Reply:
If we get the impact of snapshots proven to be minimal, then that lowers to an INFO level event.  Until proven minimal, NOTICE seems right to me.  There won't be dozens of these per second, we should be able to discard debug logs and still have records of "trigger of potentially significant actions" as the server starts doing a different sort of work to normal, disrupting its steady-state.